File: codingBatSolutions/roundSum.java
// problem from logic-2
public int roundSum(int a, int b, int c) {
  return round10(a) + round10(b) + round10(c);
}
public int round10(int num){
  int lastDigit=0;
  do {
    lastDigit = num%10;
  }while(lastDigit>10);
  if(lastDigit>=5){
    return (10-lastDigit)+num;
  } else return num - lastDigit;
}

File: codingBatSolutions/round_sum.py
def round_sum(a, b, c):
  return round10(a) + round10(b) + round10(c)
  
def round10(n):
  if n<10 and n>5:
    return 10
  if n%10 < 5:
    return n-(n%10)
  else:
    return n+(10-n%10)
